src/commands/until_front_of_line.yml

```diff
@@ -187,6 +187,8 @@ steps:
               break
             fi
           else
+            # If we fail, reset confidence
+            confidence=0
```

Comment:
Sorry for the delay I meant to respond and blanked. line 177 makes great sense, thank you.

I don't understand why line 191 (confidence=0) is needed since the confidence should never be raised by line 182 to reach this point.

it feels like adjusting the rules for confidence should be separate disccussion, and something that could probably be improved altogether.

Reply:
Thanks for merging the change, glad you found it helpful!